src/pages/SubmissionConfig.js

```diff
@@ -26,44 +33,57 @@ export const SubmissionConfig = ({ methodId }) => {
   const [runSetName, setRunSetName] = useState()
   const [runSetDescription, setRunSetDescription] = useState()
 
+  // TODO: this should probably be moved to a scope more local to the data selector
+  const [sort, setSort] = useState({ field: 'name', direction: 'asc' })
+
   const signal = useCancellation()
 
-  useOnMount(() => {
-    const loadMethodsData = async () => {
```

Comment:
these next few changes pull out the `load<X>Data` methods out of the `useOnMount` call. I did this because `loadRecordsData` needs to be outside of this scope, and thought keeping some inside and some outside of `useOnMount` signaled a distinction that doesn't really exist (basically, I think it's slightly more understandable this way)